SpatialCellSize.ipynb

```
@@ -0,0 +1,158 @@
+    "from skopt import gp_minimize\n",
+    "from skopt import Optimizer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read the census block extents."
+   ]
+  },
+  {
```

Ruff (0.16.0)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)